src/templates/blog-post.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Link, graphql} from 'gatsby';
+
+import DiscussionLink from '../components/discussion-link.js';
+import ShortBio from '../components/blog-bio/short-bio';
+import Bio from '../components/blog-bio/bio';
+import Layout from '../components/layout/layout';
+import Seo from '../components/seo';
+
+import * as styles from './blog-post.module.scss';
+import './prism.css';
+import './prism.js';
+
+const BlogPostTemplate = ({data, location}) => {
+  const post = data.markdownRemark;
+  const {previous, next} = data;
+  const author = post.frontmatter.author;
+  const avatar = (
+    author.gifPlaceHolder.childImageSharp.gatsbyImageData
+  );
+
+  return (
+    <Layout location={location}>
+      <Seo
+        title={post.frontmatter.title}
+        description={post.frontmatter.description || post.excerpt}
+      />
+      <article
+        itemScope
+        itemType='http://schema.org/Article'
+      >
+        <div className={styles.postContainer}>
+          <header className={styles.header}>
+            <h1 itemProp='headline'>{post.frontmatter.title}</h1>
+            <div className={styles.timeAuthorRow}>
+              <time datetime={post.frontmatter.date}>{post.frontmatter.dateReadable}</time>
+              <ShortBio fullName={author.fullName}/>
+            </div>
+          </header>
+          <section
+            className={styles.postContent}
+            dangerouslySetInnerHTML={{__html: post.html}}
+            itemProp='articleBody'
+          />
+          <section
+            className={styles.discussLinks}
+          >
+            <DiscussionLink linkType='LinkedIn' link={post.frontmatter.linkedin_link}/>
+            <DiscussionLink linkType='Hacker News' link={post.frontmatter.hackernews_link}/>
+          </section>
+        </div>
+        <footer className={styles.footer}>
+          <div className={styles.bioFooter}>
+            <div className={styles.postContainer}>
+              <Bio
+                fullName={author.fullName}
+                writtenDate={post.frontmatter.dateReadable}
+                avatar={avatar}
+                email={author.email}
+                github={author.github}
+                linkedIn={author.linkedIn}
+              />
+            </div>
+          </div>
+          <div className={styles.navFooter}>
+            <div className={styles.postContainer}>
+              <Link to="/blog">Continue Reading</Link>
```

Comment:
Reckon `Continue reading` is more natural (than with caps `R`)?

Reply:
ae5ddaa